src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,17 +1,44 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
 import React from 'react';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions as filterActions } from '../../features/filter';
+import { Status } from '../../types/Status';
+import { FilterByParameters } from '../../utils/filterByParameters';
 
 export const TodoFilter: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const query = useAppSelector(state => state.filter.query);
+  const status = useAppSelector(state => state.filter.status);
+
+  const updateQuery = (value: string) => {
+    dispatch(filterActions.setQuery(value));
+  };
+
+  const updateStatus = (value: Status) => {
+    dispatch(filterActions.setStatus(value));
+  };
+
   return (
     <form
       className="field has-addons"
       onSubmit={event => event.preventDefault()}
     >
       <p className="control">
         <span className="select">
-          <select data-cy="statusSelect">
-            <option value="all">All</option>
-            <option value="active">Active</option>
-            <option value="completed">Completed</option>
+          <select
+            data-cy="statusSelect"
+            value={status}
+            onChange={({ target }) => updateStatus(target.value as Status)}
+          >
+            <option value="all">
+              {FilterByParameters.All}
+            </option>
+            <option value="active">
+              {FilterByParameters.Active}
+            </option>
+            <option value="completed">
+              {FilterByParameters.Copleted}
```

Comment:
There is a typo in the constant name, change it to 'FilterByParameters.Completed'